### PR TITLE
adding support in command line for xm|xl

### PR DESCRIPTION
**Pull request by mag009**
## Pull Request Checklist

**Is this in reference to an existing issue?**
#### General
- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
- [x] Update README with any necessary configuration snippets
- [x] Binstubs are created if needed
- [x] RuboCop passes
- [x] Existing tests pass 
#### New Plugins
- [ ] Tests
- [ ] Add the plugin to the README
- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: bin/metrics-xen.rb [hunk 1]
```diff
@@ -29,6 +29,12 @@
 require 'socket'
 
 class XenGraphite < Sensu::Plugin::Metric::CLI::Graphite
+  option :info,
```
Comment (eheydrick) [inline bin/metrics-xen.rb]: can you rename this option to `info_command` so it's more clear what it is.

**Comment by eheydrick** [comment]
Left a comment, otherwise looks good. Thanks!

**Comment by mag009** [comment]
there you go, thanks!

File: bin/metrics-xen.rb [hunk 2]
```diff
@@ -29,6 +29,12 @@
 require 'socket'
 
 class XenGraphite < Sensu::Plugin::Metric::CLI::Graphite
+  option :info_command,
+         short: '-c xm',
+         long: '--cmd xm',
+         description: 'xm|xl',
+         default: 'xl'
```
Comment (sstarcher) [inline bin/metrics-xen.rb]: the previous default was xm

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,9 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## Unreleased]
 
+## [0.0.4] - 2016-05-23
```
Comment (majormoses) [inline CHANGELOG.md:8]: remove version we will bump on release.

File: bin/metrics-xen.rb (hunk 2 above)
Comment (majormoses) [reply bin/metrics-xen.rb]: please change to keep this backwards compatible.

**Review by majormoses — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (mag009) [reply CHANGELOG.md:8]: sure no problem

File: bin/metrics-xen.rb (hunk 2 above)
Comment (mag009) [reply bin/metrics-xen.rb]: you mean change the default to xm?

**Review by mag009 — COMMENTED** [review]
made the changes requested cheers

**Comment by majormoses** [comment]
@mag009 
I checked and the issue is only with ruby < 2 which we dont support, I am ok with merging this and I will need to remove testing for 2.x before we can release (it wont let me)

**Review by majormoses — APPROVED** [review]
CI issues are ruby 1.9 and I will fix prior to release.

**Pull request merged**